ui/js/filters.js

```diff
@@ -144,3 +147,61 @@ treeherder.filter('alertStatus', [
 treeherder.filter('encodeURIComponent', function() {
     return window.encodeURIComponent;
 });
+
+/*
+ * Given a bug id, fetch the status and summary from
+ * bugzilla for use in tooltips on links to that bug.
+ */
+function initMouseOver(bugLink) {
+    var bugsCacheEntry = bugsCache[bugLink.getAttribute("data-bugid")];
+    if(bugsCacheEntry) {
+        setTitleAndClearListeners(bugLink, bugsCacheEntry);
+    } else {
+        hoverTimeout = window.setTimeout(getBugInfo, 1000, bugLink);
+    }
+}
+
+function cancelMouseOver() {
+    window.clearTimeout(hoverTimeout);
+}
+
+function getBugInfo(bugLink) {
+    var bugID = bugLink.getAttribute("data-bugid");
+    var bugURL = '/api/bugzilla/' + bugID + '/get_bug/?include_fields=status,summary';
```

Comment:
So if we're going to query bugzilla directly, I think you can just replace this line with:

```js
var bugURL = 'https://bugzilla.mozilla.org/rest/bug/' + bugID + '/get_bug/?include_fields=status,summary';
```

and it should just work. This has the additional bonus of letting you test any changes to the feature completely on the client side.